src/commands/service/create.ts

```diff
@@ -41,15 +41,23 @@ export interface CommandOptions {
   packagesDir: string;
   pathPrefix: string;
   pathPrefixMajorVersion: string;
+  serviceBuildVg: string;
+  serviceBuildVariables: string;
 }
 
 export interface CommandValues extends CommandOptions {
   k8sPort: number;
   middlewaresArray: string[];
   ringNames: string[];
   variableGroups: string[];
+  serviceVgArray: string[];
+  serviceVariablesArray: string[];
 }
 
+export const validUpperUnderscoreCase = (segment: string): boolean => {
+  return !!segment.match(/^[A-Z0-9_]+$/);
```

Comment:
This pattern will allow a single underscore `_` to be a valid value. Is that considered valid?

Reply:
I just tested, and using multiple underscores appear to be valid. See here: https://github.com/yradsmikham/bedrock-cli/blob/e1dcc679330e2c4a1486e5eec6214ea3c951f6c5/src/commands/service/create.test.ts#L177